lang/python/python-cryptography/patches/0004-make-rust-completely-optional-during-build.patch

```diff
@@ -0,0 +1,33 @@
+--- a/setup.py
++++ b/setup.py
+@@ -11,7 +11,7 @@ import sys
+ from setuptools import find_packages, setup
+ 
+ try:
+-    from setuptools_rust import RustExtension
++    pass
+ except ImportError:
+     print(
+         """
+@@ -43,9 +43,9 @@ with open(os.path.join(src_dir, "cryptog
+ # `pyproject.toml`
+ setuptools_rust = "setuptools-rust>=0.11.4"
+ install_requirements = ["cffi>=1.12"]
+-setup_requirements = install_requirements + [setuptools_rust]
++setup_requirements = install_requirements
+ 
+-if os.environ.get("CRYPTOGRAPHY_DONT_BUILD_RUST"):
++if True:
+     rust_extensions = []
+ else:
+     rust_extensions = [
+@@ -129,9 +129,6 @@ try:
+                 "twine >= 1.12.0",
+                 "sphinxcontrib-spelling >= 4.0.1",
+             ],
+-            "sdist": [
+-                setuptools_rust,
+-            ],
```

Comment:
I think these lines can be left in `extras_require` as `setuptools_rust` is just a string (`"setuptools-rust>=0.11.4"`) and `sdist` is (should?) never specified as an install option.

Reply:
ack